node/block.go

```diff
@@ -58,7 +58,7 @@ func (node *Node) processBlock(ctx context.Context, pubSubMsg pubsub.Message) (e
 	// Don't be too quick to change that, though: the syncer re-fetching the block
 	// is currently critical to reliable validation.
 	// See https://github.com/filecoin-project/go-filecoin/issues/2962
-	err = node.Syncer.HandleNewTipset(ctx, types.NewTipSetKey(blk.Cid()))
+	err = node.Syncer.HandleNewTipset(ctx, pubSubMsg.GetFrom(), types.NewTipSetKey(blk.Cid()))
```

Comment:
We include who the block is from so we can populate the PeerManager used by the syncer